tools/RAiDER/delayFcns.py

```diff
@@ -124,9 +140,53 @@ def get_delays(stepSize, pnts_file, wm_file, interpType = '3D',
 #        pass
 
 #    return wet, hydro
+    time_elapse = (time.time() - t0)
```

Comment:
The tqdm function in the above for-loop tracks time usage, so I suggest to remove the explicit time-checking.

Reply:
I think we can put the time printout in the get_delay fcn under the verbose option (or logging once we get it).

Reply:
Re Yang and my conversation, tqdm isn't relevant for the parallel for-loop, so we'll just use the start and end times but them under a if verbose flag.